config/announce.yml

```diff
@@ -1,4 +1,5 @@
 app_name: feeder
+queue_name_prefix: <%= ENV['ANNOUNCE_RESOURCE_PREFIX'] || ENV['RAILS_ENV'] %>
```

Comment:
Hrmm...

Could you instead delete this line, and change `application.rb` to:

```ruby
config.active_job.queue_name_prefix = ENV['ANNOUNCE_RESOURCE_PREFIX'] || ENV['RAILS_ENV']
```

I think you said, if you don't provide this yml config, it defaults to the active job `queue_name_prefix`, right?